.husky/pre-commit

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+branch="$(git rev-parse --abbrev-ref HEAD)"
+
+if [ "$branch" = "master" ]; then
```

Comment:
Let's check for "main" too so when we eventually change the main branch name it won't break here.